guides/coding_style/c++/style.md

```diff
@@ -40,6 +40,9 @@ Including Header Files
 ===
 When including header files, only include files that are _portable_ across all compilers (Unless where applicable). Do not include `<bits/stdc++.h>`.
 
+Please pay attention to that if the header is _not in use_, then do not include it (if only implementation use it, then include the file in .cpp, not .hpp).
```

Comment:
Reword: Pay attention to whether the header is \_not in use\_, and do not include it if so. If only the implementation uses  it, then include the file in .cpp, not .hpp.